site/about/campus-images-mobile.php

```diff
@@ -0,0 +1,20 @@
+<div class="campus-image-mobile">
+          <div class="campus-image-mobile-div">
+            <div class="campus-image-mobile-edit-1">
+              <img src=" ../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
+              <p> Space Rental Options</p>
+            </div>
+          </div>
+          <div class="campus-image-mobile-div">
+            <div class="campus-image-mobile-edit-2">
+              <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
+              <p>labs, Art Studios & Rehearsal Spaces</p>
```

Comment:
Wherever you are using `&` in your HTML you want to put `&amp;` instead. Basically, the `&` can be used in the HTML code so the browser will get confused and think it's code and not text. The `&amp;` stands for ampersand and when you view it in the browser it will show up as `&`. The same applies to `<` and `>`. Those characters are used in HTML so if you want to actually show them on the page you have to use `&lt;` and `&gt;`